### ISSUE TITLE
WCAG 2.2 Violation - There are instances of illogical focus order present within the interface.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#41]
#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.3 Focus Order](https://www.w3.org/TR/WCAG22/#focus-order)

### Instances

1. #42 

#### Further information:

1. [The "Intent" section of "Understanding Success Criterion 2.4.3: Focus Order | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html#intent)
